LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIPersonnel_SoldierListItem_LW.uc

```diff
@@ -386,13 +395,67 @@ function AddClassColumnIcons(XComGameState_Unit Unit)
 	WillValue.SetHtmlText(class'UIUtilities_Text'.static.GetColoredText(GetWillString(Unit), Unit.GetMentalStateUIState()));
 }
 
+simulated function int GetWillValueColour(XComGameState_Unit Soldier, bool Disabled, bool Focused)
+{
+	// KDM : Colour the will value text according to these, ordered, rules :
+	// 1.] If the soldier row is disabled, use the disabled, greyish, text colour.
+	// 2.] If the soldier row is focused, use black.
+	// 3.] Use green/yellow/red depending upon the mental status of the soldier associated with this row.
+	if (Disabled)
+	{
+		return eUIState_Disabled;
+	}
+	else if (Focused)
+	{
+		return -1;
+	}
+	else
+	{
+		return int(Soldier.GetMentalStateUIState());
+	}
+}
+
+simulated function string GetCombatIntelligenceColour(XComGameState_Unit Soldier, bool Disabled, bool Focused)
+{
+	// KDM : Colour the combat intelligence image according to these, ordered, rules :
+	// 1.] If the soldier row is disabled, use the disabled, greyish, text colour : 0x828282.
+	// 2.] If the soldier row is focused, use black : 0x000000.
+	// 3.] Use red/orange/yellow/green blue depending upon the soldier's combat intelligence level.
+	if (Disabled)
+	{
+		return "0x828282";
+	}
+	else if (Focused)
+	{
+		return "0x000000";
+	}
+	else
+	{
+		switch (Soldier.ComInt)
+		{
+			case eComInt_Standard:
+				return "0xbf1e2e";
+			case eComInt_AboveAverage:
+				return "0xe69831";
+			case eComInt_Gifted:
+				return "0xfdce2b";
+			case eComInt_Genius:
+				return "0x53b45e";
+			case eComInt_Savant:
+				return "0x27aae1";
+			default:
+				return "0x000000";
```

Comment:
Would you be OK to make this config vars instead? I ran out of juice and didn't manage to make that final step. If you don't have time/energy/whatever, that's OK. Just let me know if possible, thanks.

Reply:
I added the config variables within XComLW_UI.ini; this is where I have put all of my other UI config variables thus far. I used upper case, separated by underscores, like all of my other variables; I find this easy to read and differentiate from normal variables.